conanfile.py

```diff
@@ -12,17 +14,19 @@ class PyBind11Conan(ConanFile):
     no_copy_sources = True
 
     def source(self):
-        tools.get("%s/archive/v%s.tar.gz" % (self.homepage, self.version))
+        tools.get("%s/archive/v%s.tar.gz" % (self.homepage, self.upstream_version))
```

Comment:
[nitpick] use format to format the string?

Reply:
in engine team, I think we prefer the old `%` instead of the new `.format()`, it's shorter. We like `f"..."` but it wouldn't work on Python 3.5 (Ubuntu 16.04) if I remember correctly.